suite-native/accounts/src/components/AccountListItem.tsx

```diff
@@ -44,6 +49,7 @@ export const AccountListItem = ({
     onPress,
     disabled,
     hideTokens = false,
+    fiatBalance,
```

Comment:
What abount not passing this as a prop but rather select it inside `AccountListItem` with a selector?

Reply:
Because sometimes this selector will be called but value will be unused anyway so I don't want to waste resources